pkgs/applications/science/physics/elmerfem/default.nix

```diff
@@ -13,19 +13,45 @@ stdenv.mkDerivation rec {
 
   hardeningDisable = [ "format" ];
 
-  nativeBuildInputs = [ cmake gfortran pkg-config git ];
-  buildInputs = [ mpi blas liblapack qt4 qwt6_qt4 ];
+  nativeBuildInputs = [
+    cmake
+    gfortran
+    pkg-config
+    git
+    qt5.wrapQtAppsHook
+  ];
+  buildInputs = [
+    mpi
+    blas
+    liblapack
+    libsForQt5.qt5.qtbase
+    libsForQt5.qt5.qtscript
+    libsForQt5.qwt
+    libGL
+    libGLU
+    opencascade
+    vtkWithQt5
+  ];
 
   preConfigure = ''
     patchShebangs ./
```

Comment:
Can we be more precise?

Reply:
As I recall there are many files to patch, I don’t think this is an issue.

Reply:
> I don’t think this is an issue.

Well, it takes longer if the script looks at every file.